DEV-9/DEV-9/PageObjects/MailRu/PersonalDataPage.cs

```diff
@@ -0,0 +1,56 @@
+﻿using OpenQA.Selenium;
+
+namespace DEV_9.PageObjects.MailRu
+{
+    /// <summary>
+    /// The personal data page.
+    /// </summary>
+    public class PersonalDataPage
+    {
+        /// <summary>
+        /// The WebDriver.
+        /// </summary>
+        private IWebDriver driver;
+
+        /// <summary>
+        /// The nickname box.
+        /// </summary>
+        public IWebElement NicknameBox => this.driver.FindElement(By.XPath("//input[@id='NickName']"));
+
+        /// <summary>
+        /// The submit button.
+        /// </summary>
+        public IWebElement SubmitButton => this.driver.FindElement(By.XPath("//span[text() = 'Сохранить']"));
```

Comment:
if my browser has another locale? It is not a good way to be attached to the text